src/utils/utils_cleaner.c

```diff
@@ -837,8 +837,9 @@ void ocf_cleaner_sort_flush_containers(struct flush_container *fctbl,
 	int i;
 
 	for (i = 0; i < num; i++) {
-		ocf_cleaner_sort_flush_data(fctbl[i].flush_data,
-				fctbl[i].count);
+		if (fctbl[i].count > 0)
```

Comment:
If there is only one element, sorting is not necessary, so technically, the condition could be `fctbl[i].count > 1` 😆